### PR TITLE
feat(ui): add actions bar with default setup

### PR DESCRIPTION
**Pull request by BleedingDev**
This is default look of action bar. It has Copy and Download buttons.

image

### CODE REVIEW

File: libs/ui/design-system/src/lib/copy-wrapper.tsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+import { toast } from 'sonner'
+
+type Position = 'top-right' | 'top-left' | 'bottom-right' | 'bottom-left'
+
+interface CopyWrapperProps {
+  showCopyButton?: boolean
+  position?: Position
+  onCopySuccess?: () => void
+  onCopyError?: (error: Error | string) => void
+  className?: string
```
Comment (devodii) [inline libs/ui/design-system/src/lib/copy-wrapper.tsx:14]: I believe Mixins would be a great way to compose the props in the future.

File: libs/ui/design-system/src/lib/copy-wrapper.tsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    'top-left': `${base} top-2 left-2`,
+    'bottom-right': `${base} bottom-2 right-2`,
+    'bottom-left': `${base} bottom-2 left-2`,
+  }
+  return positions[position]
+}
+
+export const CopyWrapper: FC<PropsWithChildren<CopyWrapperProps>> = ({
+  children,
+  showCopyButton = true,
```
Comment (devodii) [inline libs/ui/design-system/src/lib/copy-wrapper.tsx:31]: what's the purpose of this prop?
what other way do we get the user to copy stuff?

**Review by devodii — APPROVED** [review]
LGTM.

File: libs/ui/design-system/src/lib/copy-wrapper.tsx (hunk 2 above)
Comment (BleedingDev) [reply libs/ui/design-system/src/lib/copy-wrapper.tsx:31]: It was just fast iteration, I was about to get rid of it. :)

File: libs/ui/design-system/src/lib/copy-wrapper.tsx (hunk 1 above)
Comment (BleedingDev) [reply libs/ui/design-system/src/lib/copy-wrapper.tsx:14]: Yes, but... it should be done in one iteration everywhere. :)

Comment (devodii) [reply libs/ui/design-system/src/lib/copy-wrapper.tsx]: Sounds good

**Pull request merged**